cms/bundles/management/commands/publish_bundles.py

```diff
@@ -117,21 +56,19 @@ def _handle_bundle_action(self, bundle: Bundle) -> None:
 
             # Confirm the bundle is still approved
             if bundle.status != BundleStatus.APPROVED:
-                logger.error("Bundle no longer approved", extra={"bundle_id": bundle.id})
+                logger.error("Bundle no longer approved", extra={"bundle_id": bundle.pk})
                 return
 
             self.handle_bundle(bundle)
         except Exception:  # pylint: disable=broad-exception-caught
-            logger.exception("Publish failed", extra={"bundle_id": bundle.id, "event": "publish_failed"})
+            logger.exception("Publish failed", extra={"bundle_id": bundle.pk, "event": "publish_failed"})
```

Comment:
Out of scope, but it would be good to at least send an email to the relevant people when this happens. (preview team/ last editor/ person who approved?)

Potentially in the future also a visual notification within Wagtail (another panel on the the main dashboard?)